packages/calling/src/Voicemail/UcmBackendConnector.ts

```diff
@@ -254,10 +260,11 @@ export class UcmBackendConnector implements IUcmBackendConnector {
     });
     const contentInfo = response?.body as UcmVMContentResponse;
     const respHeaders = response.headers;
+    const statusCode = response.statusCode;
     const mediaType = respHeaders?.mediatype as string;
     const mediaContent = contentInfo as string;
     const responseDetails = {
-      statusCode: response.statusCode as number,
+      statusCode: statusCode as number,
```

Comment:
` as number` has known to be problematic and we are changing it whenever we encounter it during new changes.
```suggestion
      statusCode: Number(statusCode),
```

Reply:
Done